classes/DirtEvent.sc

```diff
@@ -37,7 +37,9 @@ DirtEvent {
 	splitName {
 		var s, n;
 		#s, n = ~sound.asString.split($:);
-		if(~bank.notNil) { s = ~bank ++ s };
+		if(~bank.notNil) {
```

Comment:
I think splitName isn't called if `s` is defined, and tidal sends `s` rather than `sound`, so this isn't reached.

Reply:
ah that's true. I'll have to do it a little difefrently then.